### PR TITLE
Code optimization

### PR DESCRIPTION
**Pull request by PijusOz**

### CODE REVIEW

File: back-end-side/Models/Record.cs [hunk 1]
```diff
@@ -17,5 +17,16 @@ public class Record
...
+
+        public int CompareTo(object? obj)
+        {
+            if (obj == null) return 1;
+
+            Record nextRecord = obj as Record;
+            if (nextRecord != null)
+                return this.Date.CompareTo(nextRecord.Date);
+            else
+                throw new ArgumentException("exeption");
```
Comment (Vaivaal) [inline back-end-side/Models/Record.cs:29]: Exeption should be clearer, not just "exeption"

File: front-end-side/src/Pages/ExpensesPages.js [hunk 2]
```diff
@@ -44,27 +44,8 @@ const ExpensesPages = ({ categoryType }) => {
...
-                                inputData.data.map((item) => {
-                                    const { date, seller, details, amount, category } = item;
-                                    if (category == categoryType)
-                                        return (
-
-                                            <tr>
-                                                <td>{date.slice(0, 10)}</td>
-                                                <td>{seller.slice(0, 37)}</td>
-                                                <td>{details.slice(0, 37)}</td>
+                                                <td>{seller}</td>
```
Comment (dissco6) [inline front-end-side/src/Pages/ExpensesPages.js:47]: If you remove slice function here, then remove them in ``Data.js`` as well

File: back-end-side/Models/Record.cs (hunk 1 above)
Comment (Vaivaal) [inline back-end-side/Models/Record.cs:29]: exception should be clearer, not just "exception"

File: front-end-side/src/Pages/Data.js [hunk 3]
```diff
@@ -45,44 +45,6 @@ const EnterData = () => {
                 setInputValues(data);
             });
```
[the remark this answers is not in the document]
Comment (PijusOz) [reply front-end-side/src/Pages/Data.js:46]: Okily dokily

File: front-end-side/src/Pages/ExpensesPages.js (hunk 2 above)
Comment (PijusOz) [reply front-end-side/src/Pages/ExpensesPages.js:47]: Aha

File: back-end-side/Models/Record.cs (hunk 1 above)
Comment (PijusOz) [reply back-end-side/Models/Record.cs:29]: K.

**Pull request merged**